sale_request/wizards/create_sale_order_wizard_view.xml

```diff
@@ -6,6 +6,9 @@
         <field name="arch" type="xml">
             <form>
                 <div class="alert alert-danger" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('has_lines', '=', True),('on_time', '=', True)]}">
+                    <bold>Sorry!</bold>, Do you want to confirm out of schedule?
+                </div>
+                <div class="alert alert-danger" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('on_time', '=', False)]}">
```

Comment:
I think you need to validate also has_lines because it will be shown even when the order has lines and are on time.

What do you think?